### DISCUSSION TITLE
Maybe can make a charge timer

### DISCUSSION THREAD

**Discussion by NinewordsLee** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#824] [Ideas]
Sometimes I wanna my laptop charge in 100% when I need to go out for a day.
But I usually charge it before I sleep, so I need to set it charge 100% very early.
And the laptop will stay 100% over 7 hours, which is not good.

If it have a timer can pause the charging, and charge it automatically to 100% just on my setting time, it will be perfect!

**Comment by MatthiasKerbl** [comment]
Hi @NinewordsLee,

Thank you for your input. We already have a couple of feature requests about this open and we will see what we can do. However, our capabilities might be limited, since macOS does not allow third-party apps to run while in sleep.

Cheers,
Matthias

**Reply by NinewordsLee** [reply]
Thanks for your reply, I will look forward for this, hope it will have some solution can fix macOS limit.
